Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/Faq.rst

```diff
@@ -45,6 +45,24 @@ of the extension or the :ref:`crowdin-initiative`.
 
    The language needs to be supported by TYPO3 itself as well, see :ref:`i18n_languages` for a list of all languages.
 
+.. _crowdin-faq-language-xlf-format:
+
+How to convert to the new language xlf file format
```

Comment:
By first reading I thought you mean the conversion from XLIFF version 1.0 to 1.2. This gets only clear when reading the replacement later. This should be made more clear, that "only" the adding of the "resname" attribute is meant.

Reply:
ok